### ISSUE TITLE
About pretrained weights and dataset for panel segmentation.

### ISSUE THREAD

**Issue by charles-pyj** [GaetanLepage/panel-seg#1]
Hi, I am Yijun Pan, currently a upcoming senior at University of Michigan major in Data Science. Recently I am conducing a research into training multimodal models and it requires me to segment medical images. Would you kindly share pre-trained model weights or dataset you collected? It would be nice of you to provide some guidance on how you collected those data, too. Thanks!

**Comment by GaetanLepage** [comment]
Hi,
Thank you for showing interest in my project. Unfortunately, although most of the pipeline is functional, I was not able to get decent enough performance on my trainings.
Also, since then, several very good works have been published on this topic:
- [Compound Figure Separation of Biomedical Images: Mining Large Datasets for Self-supervised Learning](https://arxiv.org/pdf/2208.14357)
- [Compound Figure Separation of Biomedical Images with Side Loss](https://arxiv.org/pdf/2107.08650)
- [EXSCLAIM! – An automated pipeline for the construction of labeled materials imaging datasets from literature](https://arxiv.org/pdf/2103.10631)
...

Hence, I would rather encourage you to have a look at those more advanced techniques.
Of course, if you have some questions or if you want to have a discussion about my method, please do not hesitate to reach out !

Best of luck with your research.
